.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "semi": false,
+  "printWidth": 80
```

Comment:
Hmm seems that I didn't saved my comment here.
I think we can leave out `"printWidth": 80` as it's the default.